tests/Bootstrap/ListGroupTest.elm

```diff
@@ -0,0 +1,210 @@
+module Bootstrap.ListGroupTest exposing (..)
+
+import Bootstrap.ListGroup as ListGroup
+import Html exposing (text)
+import Html.Attributes as Attr exposing (href)
+import Test exposing (Test, test, describe)
+import Expect
+import Test.Html.Query as Query
+import Test.Html.Selector as Selector exposing (tag, class, classes, attribute, disabled)
+
+
+{-| @ltignore
+-}
+all : Test
+all =
+    Test.concat
+        [ vanillaListGroup
+        , keyedListGroup
+        , contextualListGroup
+        , customListGroup
+        , contextualListGroup
+        ]
+
+
+vanillaListGroup : Test
+vanillaListGroup =
+    let
+        html =
+            ListGroup.ul
+                [ ListGroup.li [] [ text "List item 1" ]
+                ]
+
+        active =
```

Comment:
Why not just put an active item and disabled item in the list for `html` and then have assertions for 0, 1, 2 indexed items ?